modules/ROOT/pages/create-keystore.adoc

```diff
@@ -0,0 +1,45 @@
+= Mulesoft Keystore for Endpoints
+
+== Keystore Settings
+
+Using the Mulesoft Keystore, the host xref:Certificates.adoc [certificates] and private key can be stored and reused for endpoint generation.
```

Comment:
Please rephrase this as:
You can create a new keystore for the Host organization when configuring AS2 receive and send endpoints, and reuse it in new endpoint configurations by selecting an existing keystore configuration.